juntagrico/admins/__init__.py

```diff
@@ -76,7 +76,7 @@ class AreaCoordinatorBaseMixin(BaseModelAdmin):
     def _has_permission(self, request, obj=None, access=None):
         if access is None or access in self.coordinator_permissions:
             area = {'area': self.get_area(obj)} if obj else {}
-            return request.user.member.area_access.filter(**area, **{self.coordinator_access: True}).exists()
+            return request.user or False and request.user.member.area_access.filter(**area, **{self.coordinator_access: True}).exists()
```

Comment:
What is this change for? Can we arrive here without a user in the request? 
It should be `(request.user or False) and ...` with brackets if we need it.

Reply:
Yes, in releaes/2.0 an anonymous request (no login) to example.com/admin/ leads to an error instead of the login form. the above is a fix for this behavior. will add the brackets for readability.